### DISCUSSION TITLE
Error on opening notifications

### DISCUSSION THREAD

**Discussion by efb4f5ff-1298-471a-8973-3d47447115dc** [community/community#35590] [Mobile]
So at night i turn my phone off and receive this error. I noticed a pattern for a workaround for this issue but of course i don't want to do that every time and just work as normally

https://user-images.githubusercontent.com/73130443/194755202-f77ffcfa-37fd-4ef7-9ff3-0a9e43b66a18.mp4

Turn phone on -> open notifications tab im github app -> see error -> refesh to check if phone is not acting up -> opening my laptop and logging into github in the browser -> github notices that there is activity on my account -> refresh on phone -> see error is gone

Edit: this also happens when my phone isnt turned off. Seems like it just throws this error because of X hours of inactivity

**Comment by iskae** [comment]
Hi @efb4f5ff-1298-471a-8973-3d47447115dc 👋 

Thanks so much for reaching out and also attaching a video ❤️ With that video we were able to track down the problem 🙇 

An internal issue has been created for this and we will be investigating this further 🙇 Please stay tuned for upcoming releases and hopefully we can solve this issue 😊 

Cheers!

**Reply by onesounds** [reply]
<img src="https://user-images.githubusercontent.com/6903107/197696646-d9199e6a-b052-4a54-89b5-0d818089f71e.jpg" height="300">

same here. And also can't check the some PR by repo/pr menu too. It is not happen all PR. It happen just some PRs. (I guess the PR was recently showed in notifications)
